.gitignore

```diff
@@ -0,0 +1,3 @@
+/target
+.idea/
+Cargo.lock
```

Comment:
Since this will generate a binary, I don't think we want to exclude `Cargo.lock` anymore.

Reply:
Updated